src/App.js

```diff
@@ -7,35 +7,35 @@ import FingerprintJS from "@fingerprintjs/fingerprintjs-pro";
 // You can also simply whitelist the addresses from which you will be making the calls in your user dashboard
 // For the server API key, we reccommend setting up an Auth Header instead, you can read more in the docs:
 // https://dev.fingerprintjs.com/docs/server-api
-const BROWSERAPIKEY = "P7vQXeD8G1VlmDAwwFir";
-const SERVERAPIKEY = "9MLmWT6ziZqePf9JrDZP";
-
-// This will limit the amount of visits the api queries
-const limitTo = 10;
+const BROWSER_API_KEY = "k6mBs3JggSu0q48OS7yz";
+const SERVER_API_KEY = "iJhQgk9AG5w91l0bmSH1";
 
 export default function App() {
+    // Keeping track of if the visitorId has loaded yet
+    const [isLoading, setIsLoading] = useState(true);
     // Visitor ID is the unique string ID used to identify a visitor of the site.
-    const [visitorId, setVisitorId] = useState("Waiting...");
-    // Server Data is the returned data from the SERVER API call.
-    const [serverData, setServerData] = useState("");
+    const [visitorId, setVisitorId] = useState("");
+    // vistorHistory is the returned data from the SERVER API call.
+    const [visitorHistory, setVisitorHistory] = useState("");
     // This is just to display the response title (underneath the button)
     const [responseSummary, setResponseSummary] = useState("");
 
-    // This will get the Visitor ID on page load
+    // This will get the Visitor ID on component mount
     useEffect(() => {
         FingerprintJS.load({
-            token: BROWSERAPIKEY,
+            token: BROWSER_API_KEY,
         })
             .then((fp) => fp.get())
             .then((result) => {
                 setVisitorId(result.visitorId);
+                setIsLoading(false);
             });
     }, []);
 
     // this will query the server API for the visitor history.
-    const callServerAPI = () => {
+    const getVisits = (limitTo) => {
```

Comment:
this is a super minor thing but usually when a method name starts with `get`, it is expected to return some value. I think it would be cleaner to have a click handler function where you would get the data from the server (end method at `return response.json()`) and then set it to the state

Reply:
Yeah this makes a lot of sense. That way its more modular. thanks